core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -151,45 +151,43 @@ var TopicEditorPage = function() {
   };
 
   this.publishTopic = async function() {
-    await publishTopicButton.click();
+    await action.click('Publish Top Button', publishTopicButton);
     await waitFor.invisibilityOf(
       publishTopicButton, 'Topic is taking too long to publish.');
   };
 
+
   this.expectNumberOfQuestionsForSkillWithDescriptionToBe = async function(
       count, skillDescription) {
-    await waitFor.elementToBeClickable(
-      selectSkillDropdown, 'Skill select dropdown takes too long to appear.');
-    await selectSkillDropdown.click();
-    await element(by.css('option[label="' + skillDescription + '"]')).click();
+    await action.click('Select Skill Dropdown', selectSkillDropdown);
+    var skillDescription = element(by.css(
+      'option[label="' + skillDescription + '"]'));
+    await action.click('skill Description', skillDescription);
     await waitFor.visibilityOf(
       questionItems.first(), 'Question takes too long to appear');
```

Comment:
This is outside the scope of #10798, but in reviewing this code I realized that this check is incorrect for the case where `count === 0`. If you want, you could add an `if` check here so we only call `waitFor` when `count` is nonzero